src/auth/reducers/index.js

```diff
@@ -1,16 +1,21 @@
 import * as actions from '../actions';
 
-const initialState = {
-    isLoggedIn: null,
-};
+const initialState = { authenticated: false };
 
 export function auth(state = initialState, action) {
     switch (action.type) {
         case actions.LOGIN_USER_FULFILLED:
             return {
                 ...state,
-                isLoggedIn: true,
+                authenticated: true,
```

Comment:
oh, it already is here. Then I don't see a point in storing this information in localStorage

Reply:
Yea it is there, but when refreshing it will be lost, I will handle `redux-persist` for that.